README.md

```diff
@@ -5,7 +5,7 @@ Template project for Java 8 web app stack using spark, redis, bootstrap, jade an
 
 1. clone me
 2. install and start redis
-3. install browserify using npm -g browserify
+3. run npm install --save
```

Comment:
You don't need the --save. I wasn't really thinking and is a left over of how I added the dependencies to package.json, as you said.